src/components/coinsList/content/CoinList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import {
+  CoinListContainer,
+} from "../styles/coinListContainer_style";
+import { CoinInterface } from "types";
+import { CoinListProps } from "./coinList_types";
+import { Spin } from "antd";
+import CoinItem from "@components/coinItem/content/CoinItem";
+
+export const CoinList: React.FC<CoinListProps> = ({ data, loading }) => {
+  return (
+    <CoinListContainer>
+      {loading ? (
+        <Spin />
+      ) : (
+        data &&
+        data.map((coin: CoinInterface) => {
+          const { change, iconUrl, name, price, uuid,sparkline } = coin;
+          return (
+            <CoinItem
+              key={uuid}
+              uuid={uuid}
+              change={Number(change)}
+              iconUrl={iconUrl}
+              name={name}
+              price={price}
+              sparkline={sparkline}
+            />
```

Comment:
you dont need to pass each of them with same key and value just use spread syntax

Reply:
resolved